gradle.properties

```diff
@@ -1,3 +1,3 @@
 // Project properties
-version=3.0.0
+version=3.0.1-BETA
```

Comment:
We used to call Preview for older agent versions before GA, and then increment.  what would be the version name for next preview version? 3.0.2-BETA?

Reply:
3.0.1-BETA.2